app/bundles/AssetBundle/Controller/AssetController.php

```diff
@@ -317,10 +315,6 @@ public function newAction($entity = null)
         $page   = $session->get('mautic.asset.page', 1);
         $action = $this->generateUrl('mautic_asset_action', ['objectAction' => 'new']);
 
-        // Get upload folder
-        $uploaderHelper = $this->container->get('oneup_uploader.templating.uploader_helper');
-        $uploadEndpoint = $uploaderHelper->endpoint('asset');
```

Comment:
@mabumusa1 We'll have to revert this. Assets are broken by this change. Why did you change it?

Reply:
I don't get it. How is it related to the templating helper? The original code was creating a route. This new one is rendering a template.

Reply:
@escopecz you are right, the change is wrong. I will revert it now

Reply:
Resolved in commit 8053ea388de956e906845722136fac74a9d7fb40